app/code/Dss/ChangeCustomerPassword/composer.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "dss/m2-change-customer-password-admin",
+    "description": "Change customer password from admin side like old fashion way Magento",
+    "type": "magento2-module",
+    "version": "0.0.1",
+    "authors": [
+      {
+        "name": "Extension Team",
+        "email": "info@digitsoftsol.com",
+        "homepage": "https://www.digitsoftsol.com"
+      }
+    ],
+    "license": [
+      "OSL-3.0",
+      "AFL-3.0"
+    ],
+    "require": {
+      "php": "^8.1",
+      "magento/product-community-edition": "^2.4.5",
+      "dss/m2-core": "^0.0.1",
+      "dss/m2-seo-core": "^0.0.1"
```

Comment:
where it's used "dss/m2-seo-core": "^0.0.1"  ??

Reply:
not used this is my mistake removed